data/projects.yml

```
@@ -1,7 +1,7 @@
 title: OpenRail Projects
 introduction: |
   These are the Open Source projects which are officially hosted by the
-  OpenRail Association. They have been accepted through the <a href="https://github.com/OpenRailAssociation/technical-committee/blob/main/incubation-process.md">incubation process</a>.
+  OpenRail Association. They have been accepted through the <a href="https://github.com/OpenRailAssociation/technical-committee/blob/main/docs/incubation/process/index.md">incubation process</a>.
```

Could you please use the live link on the website?

```suggestion
  OpenRail Association. They have been accepted through the <a href="https://projects.openrailassociation.org/docs/incubation/process/">incubation process</a>.
```